src/QVariant.zig

```diff
@@ -7,6 +7,7 @@ pub const QVariant = struct {
     pub fn create(value: anytype) QVariant {
         var vptr = switch (@typeInfo(@TypeOf(value))) {
             .Null => dos.dos_qvariant_create(),
+            .Pointer => dos.dos_qvariant_create_string(value),
```

Comment:
Dirty hack to at least get QVariants of type string working

This needs to be fixed though, because its not a valid assumption